README.yaml

```diff
@@ -35,50 +35,110 @@ badges:
-    url: "https://www.terraform.io/docs/registry/modules/publish.html#requirements"
-  - name: "Terraform `random_integer` Resource"
-    description: "The resource random_integer generates random values from a given range, described by the min and max attributes of a given resource."
-    url: "https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/integer"
   - name: "Terraform Version Pinning"
     description: "The required_version setting can be used to constrain which versions of the Terraform CLI can be used with your configuration"
     url: "https://www.terraform.io/docs/configuration/terraform.html#specifying-a-required-terraform-version"
+  - name: "Spacelift"
+    description: "The most flexible CI/CD for Terraform"
+    url: "https://spacelift.io/"
+  - name: "Spacelift Documentation"
+    description: "Official documentation site for Spacelift"
+    url: "https://docs.spacelift.io/"
 
 # Short description of this project
 description: |-
-  This is `terraform-example-module` project provides all the scaffolding for a typical well-built Cloud Posse module. It's a template repository you can
+  This project provides all the scaffolding for a typical well-built Cloud Posse module. It's a template repository you can
   use when creating new repositories.
 
 # Introduction to the project
 #introduction: |-
 #  This is an introduction.
 
 # How to use this module. Should be an easy example to copy and paste.
+
 usage: |-
-  Here's how to invoke this example module in your projects
+
+  The first step is to create a new infrastructure repository in Github and link it in Spacelift. You will want the `Project root` 
+  of the infrastructure repository to point to the `top-level` project that contains the reference to this module. Beyond that,
+  you will also want to enable the `Administrative` and `Autodeploy` options in the configuration.
+
+  The `Administrative` option allows the top-level project and this module to create all other required configurations, such as stacks,
+  policies, and contexts.
+
+  Once you have the top-level project configured and ready, simply trigger it manually, or by pushing an update to the infrastructure
+  to Git.
+
+  Here's how to invoke this module in your project:
 
   ```hcl
+  provider "spacelift" {}
+
   module "example" {
-    source = "https://github.com/cloudposse/terraform-example-module.git?ref=master"
-    example = "Hello world!"
+    source = "https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation.git?ref=master"
+
+    config_file_path = "config""
+    branch           = "main"
+    repository       = "my-infra-repo"
+
+    manage_state = false
   }
   ```
 
+  **NOTE:** Be sure to set the `manage_state` value from the start, otherwise changing this value after deploying a stack may cause resources to be orphaned!
```

Comment:
```suggestion
  **IMPORTANT:** Be sure to set the `manage_state` value from the start, otherwise changing this value after deploying a stack may cause resources to be orphaned!
```